dream-journal/src/components/toolbar/toolbar.module.css

```
@@ -0,0 +1,18 @@
+.toolbar {
+  display: flex;
+  gap: 1rem;
+
+  margin-block: 1rem;
+
+  :first-child {
+    flex: 1;
+  }
```

این الان تمام first-childها رو انتخاب میکنه
بهتره از Child Combinator استفاده کنید 

```suggestion
  > :first-child {
    flex: 1;
  }
```

به قسمتِ «Child Combinator​​» در لینک زیر مراجعه کنید:
https://docs.codective.ir/guides/clean-code/common-mistakes